### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Pedro Alves da Silva & Gonçalo Carvalheiro Heleno

### DISCUSSION THREAD

**Discussion by PedroAS7** [heig-vd-dai-course/heig-vd-dai-course#461] [Show and tell]
A multiplayer game based on the American TV game "Wheel of Fortune".

# Game mechanisms

![](https://www.kcrw.com/news/shows/press-play-with-madeleine-brand/russia-self-driving-car-ryan-seacrest/wheel-fortune/@@images/rectangular_image/page-header?v=1688065790.32)

The game starts with the host showing the words or sentence to be found. The screen has all slots empty. The _n_ players gather around a wheel. The host asks a player to turn the big wheel that contains a cash prize. The contestant tries to guess a consonant. If it does exist in that puzzle, then the prize shown in the wheel is multiplied with the amount of times the consonant is in the message, and gets added to the contestant's cash pool. After getting the prize (i.e., if the player chose a consonant that exists), the contestant can either choose to choose a vowel (250$), spin again, or try to find the hidden word. This repeats for each and every contestant until the word is found.

_k_ words are to be found until the player with the most money goes to the finals. The final round will consist of a new "find the word" puzzle, where the letters R, S, T, L, N, and E are shown by default. The contestant will choose one "envelope" out of 24, and will then need to give 3 consonants and 1 vowel, until the timer starts ticking. They then have _m_ seconds to find the entire word in order to win the final prize that is inside the said envelope.

**Comment by GeraudSilvestri** [comment]
It's good for me

**Comment by lentidas** [comment]
Here it is the tag of our repo that we submit for evaluation-> https://github.com/lentidas/DAI-2425-PW2/releases/tag/v1.0.0
